src/main/java/io/stargate/sgv2/jsonapi/api/model/command/CommandStatus.java

```diff
@@ -10,9 +10,15 @@ public enum CommandStatus {
   /** The element has the count of deleted documents */
   @JsonProperty("deletedCount")
   DELETED_COUNT,
-  /** Status for reporting existing namespaces. */
+  /**
+   * Status for reporting existing namespaces. findNamespaces command is deprecated, keep this
+   * jsonProperty to support backwards-compatibility
+   */
```

Comment:
nit: consider also saying we use "namespaces" when old command name used and "keyspaces" when new one

Reply:
Thanks